src/solvers/flattening/boolbv_get.cpp

```diff
@@ -168,9 +168,11 @@ exprt boolbvt::bv_get_rec(
 
       const typet &subtype = components[component_nr].type();
 
+      const member_exprt member(
+        expr, components[component_nr].get_name(), subtype);
       return union_exprt(
```

Comment:
same here?

Reply:
As above, the `nil_exprt` is now an `index_exprt` and there can't be a `nil_exprt` reaching this point.